src/transactionsFactories/transactionsFactoryConfig.ts

```diff
@@ -59,6 +67,14 @@ export class TransactionsFactoryConfig {
         this.gasLimitEsdtNftBurn = 1000000n;
         this.gasLimitStorePerByte = 10000n;
         this.issueCost = 50000000000000000n;
+        this.gasLimitEsdtModifyRoyalties = 60000000n;
+        this.gasLimitEsdtModifyCreator = 60000000n;
+        this.gasLimitEsdtMetadataUpdate = 60000000n;
+        this.gasLimitSetNewUris = 60000000n;
+        this.gasLimitNftMetadataRecreate = 60000000n;
+        this.gasLimitNftChangeToDynamic = 60000000n;
+        this.gasLimitUpdateTokenId = 60000000n;
+        this.gasLimitRegisterDynamic = 60000000n;
```

Comment:
Breaking only in theory. In practice, in this exact context, this is perfectly fine.